.github/workflows/ubuntu.yml

```diff
@@ -21,79 +21,25 @@ jobs:
         with:
           repository: nginx/nginx
           path: nginx
-      - name: Configure nginx
+      - name: Build nginx
         working-directory: nginx
-        run: auto/configure --with-compat
-      - name: Create build directory
-        run: mkdir build
+        run: |
+          auto/configure --with-compat --with-debug --with-http_ssl_module \
+                         --with-http_v2_module --with-http_v3_module
+          make -j $(nproc)
       - name: Build module
-        working-directory: build
         run: |
-          cmake -DNGX_OTEL_NGINX_BUILD_DIR=${PWD}/../nginx/objs \
-                -DNGX_OTEL_DEV=ON ..
-          make -j 4
-          strip ngx_otel_module.so
-      - name: Archive module
-        uses: actions/upload-artifact@v4
-        with:
-          name: nginx-otel-module
-          path: build/ngx_otel_module.so
-      - name: Archive protoc and opentelemetry-proto
-        uses: actions/upload-artifact@v4
-        with:
-          name: protoc-opentelemetry-proto
-          path: |
-            build/_deps/grpc-build/third_party/protobuf/protoc
-            build/_deps/otelcpp-src/third_party/opentelemetry-proto
-  test-module:
-    needs: build-module
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v4
-      - name: Download module
-        uses: actions/download-artifact@v4
-        with:
-          name: nginx-otel-module
-          path: build
-      - name: Download protoc and opentelemetry-proto
-        uses: actions/download-artifact@v4
-        with:
-          name: protoc-opentelemetry-proto
-          path: build/_deps
-      - name: List files
-        run: ls -laR .
-      - name: Fix protoc file permissions
-        run: chmod +x build/_deps/grpc-build/third_party/protobuf/protoc
-      - name: Install perl modules
-        run: sudo cpan IO::Socket::SSL Crypt::Misc
+          mkdir build && cd build && cmake -DNGX_OTEL_NGINX_BUILD_DIR=\
```

Comment:
There is no need to obfuscate this. Let's keep each command on its own line, like it was before.

Reply:
Let me check if it works. I'm not sure because we changed the directory there.

Reply:
That works becase github uses one shell session for all these commands. Fixed.